implementations.md

```diff
@@ -7,7 +7,13 @@ This page contains an alphabetical list of projects and vendors that currently i
 **Website:** [https://elastic.co/products/apm](https://elastic.co/products/apm)
 
 **Implementations:**
-[Node.js](https://github.com/elastic/node-traceparent).
+[.NET](https://github.com/elastic/apm-agent-dotnet/blob/700754909b1ac522796294b99adcc98063efcf42/src/Elastic.Apm/DistributedTracing/TraceParent.cs),
+[Go](),
```

Comment:
```suggestion
[Go](https://github.com/elastic/apm-agent-go/blob/0365ba99e57221bfa884579c69cbcc0a16c2ded7/tracecontext.go),
```

Reply:
https://github.com/elastic/apm-agent-go/blob/master/module/apmhttp/traceheaders.go is probably the more interesting part (HTTP header parsing).